java/src/main/native/cmake/Modules/FindcuFile.cmake

```diff
@@ -0,0 +1,121 @@
+#=============================================================================
```

Comment:
We're going to need this file in libcudf as well where we should put this in a location where we can share the find module between the two.

Reply:
Do you want to do that now? I'm not sure about libcudf's timeline for integrating with cufile, so putting this in jni first and perhaps moving it later. I can move it now if you want.

Reply:
I'd like the cmake find module to be placed somewhere general now, I.E. in this folder: https://github.com/rapidsai/cudf/tree/branch-0.17/cpp/cmake

It won't be used in the general libcudf cmake yet, but it sets things up to be used directly nicely in the future while presumably still allowing the JNI build to nicely use it now.

Reply:
Done.